airgun/widgets.py

```diff
@@ -2180,8 +2180,10 @@ def fill(self, item):
         if self.header.is_displayed:
             if self.drop_down.is_displayed:
                 self.drop_down.fill(item)
-            else:
+            elif self.input_box.is_displayed:
```

Comment:
Minor nitpick: Please update docstring to also mention textarea both for class and fill function

Reply:
Updated both as you suggested.